src/main/webapp/app/exercises/shared/result/updating-result.component.ts

```diff
@@ -38,6 +38,8 @@ export class UpdatingResultComponent implements OnChanges, OnDestroy {
      */
     @Input() personalParticipation = true;
 
+    @Output() onParticipationChange = new EventEmitter<void>();
+
```

Comment:
```suggestion
    @Input()
    personalParticipation = true;

    @Output()
    onParticipationChange = new EventEmitter<void>();

```

Reply:
This is not shown in the git-diff, but there are six more usages of `@Input()` above. Not sure, if we should change that.

Reply:
It's unrelated to this change so we can keep it and stay consistent for this class. But in general I think two lines is the better style.

Reply:
I agree, but we try to keep the scope of this PR as minimal as possible due to our timely thesis deadline.